src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer.Common/BackgroundDocumentGenerator.cs

```diff
@@ -173,6 +174,7 @@ private async Task Timer_TickAsync(CancellationToken cancellationToken)
                 OnStartingBackgroundWork();
 
                 KeyValuePair<string, DocumentSnapshot>[] work;
+                List<(RazorCodeDocument Output, DocumentSnapshot Document)> results = new();
```

Comment:
Nit: Might be nice to make this a concrete record-type instead of a tuple for readability and ease of use, but it's not super important.